### PR TITLE
New auto ml tabular classification bean

### PR DESCRIPTION
**Pull request by udaypunna**
Modify Notebook: Auto ml tabular classification beans.ipynb
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/pipelines/automl_tabular_classification_beans.ipynb [hunk 1]
```diff
@@ -2,13 +2,13 @@
   "cells": [
```
Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>do not update copyright year</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>Move Dataset after Objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>remove execution output throughout notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>Use notebook template</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>use notebook template</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>Remove this text and accompanying code cell. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>use notebook template</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>consolidate imports into earlier import cell above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>I believe there is a GCPC component now for AutoML model eval</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>imports for GCPC should be defined inside of pipeline</p><p> </p><p>gcc_aip.ModelDeployOp( -&gt; _ = gcc_aip.ModelDeployOp(</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>"tabular classification_pipeline.json".replace(" ", "_") -&gt; "tabular_classification_pipeline.json"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>explain the parameters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>"tabular classification_pipeline.json".replace(" ", "_") -&gt; "tabular_classification_pipeline.json"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>remove bullet list</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/automl_tabular_classification_beans.ipynb:2]: <p>replace with deleting only the resources created in the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

**Comment by andrewferlitsch** [comment]
There are still 3 review comments to address

File: notebooks/official/pipelines/automl_tabular_classification_beans.ipynb (hunk 1 above)
Comment (krishr2d2) [reply notebooks/official/pipelines/automl_tabular_classification_beans.ipynb]: True (ModelEvaluationOp), but there are no new predictions being generated in this notebook. The evaluation metrics from the trained AutoML model are logged and used for conditioning the model deployment.

Comment (krishr2d2) [reply notebooks/official/pipelines/automl_tabular_classification_beans.ipynb]: done...

Comment (krishr2d2) [reply notebooks/official/pipelines/automl_tabular_classification_beans.ipynb]: done.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**